### PR TITLE
[Windows] .NET 5.x has been removed

### PR DESCRIPTION
**Pull request by VFedorV**
# Description
[Windows] .NET 5.x has been removed from the images

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/toolsets/toolset-2022.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
```
Comment (dmitry-shibanov) [inline images/win/toolsets/toolset-2022.json:1]: Could you please revert this change.

Comment (VFedorV) [reply images/win/toolsets/toolset-2022.json]: done
thx

**Review by ddobranic — APPROVED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Review by erik-bershel — APPROVED** [review]
